.github/workflows/pre-main.yaml

```diff
@@ -155,8 +155,9 @@ jobs:
       - name: Check out `cnf-certification-test-partner`
         uses: actions/checkout@v3
         with:
-          repository: test-network-function/cnf-certification-test-partner
+          repository: edcdavid/cnf-certification-test-partner
```

Comment:
Don't forget to change this before merging.

Reply:
fixed